src/contexts/Balances/Provider.tsx

```diff
@@ -25,43 +25,41 @@ import {
 } from 'constants/ethContractAddresses'
 
 const Provider: React.FC = ({ children }) => {
```

Comment:
My sense is these should _all_ be `BigNumber` or `strings`. Having a mixture of BN & strings adds unnecessary complexity IMO. I'm leaning towards keeping things `BigNumber`, just to make sure no one is trying to do any math on these that does not involve Big Numbers.

Reply:
I think this will also make it match the SetJS integration once we get there.